microsoft-edge/progressive-web-apps-chromium/experimental-features/index.md

```diff
@@ -0,0 +1,469 @@
+ms.prod: microsoft-edge
+keywords: microsoft edge, experiment, progressive web apps, web apps, PWAs, PWA
+---
+# Experimental features in Progressive Web Apps
+
+Microsoft Edge provides access to experimental features that are still in development.  You may test and [provide feedback](#providing-feedback-on-experimental-features) on these features to help to determine when and if to release them.  
+
+While experimental features are available in all channels of Microsoft Edge, you will get the latest experimental features using the Microsoft Edge Canary channel.  
+
+## Turn on experimental features  
+
+To turn on \(or off\) experimental features in Microsoft Edge, complete the following steps.  
+  
+1.  Open the Microsoft Edge flags webpage by navigating to `edge://flags` using the address bar. Make sure you are using a version of Microsoft Edge that has the relevant feature flag listed in this article.  Navigate to [Experimental features](#experimental-features-in-progressive-web-apps).  
+1.  Find the relevant experimental flag on the webpage and change it's status to `Enabled`. Each experiment will usually have it's own flag to toggle on or off.
+1.  Close and restart Microsoft Edge  
+
+:::image type="complex" source="../media/turn-on-experimental-flag.png" alt-text="Change an experimental flag status" lightbox="../media/turn-on-experimental-flag.png":::
+   Change an experimental flag status  
+:::image-end:::
+
+If an experimental feature doesn't have an `edge://flags` entry, instructions will be provided on how to use the command line to start Microsoft Edge with that feature specifically turned on.
+    
+> [!NOTE]
+> Experimental features are constantly being updated and may cause performance issues.  To turn off an experimental feature, open the `edge://flags` webpage and change the status of the feature to `Disabled`.  
+
+## Features Available to Test  
+
+The following sections describe the new experimental web app features that are available for testing and validation on Microsoft Edge.  
+
+| Feature | Microsoft Edge version | Platform |  
+|:--- |:--- |:--- |  
+| [URL Protocol Handling](#url-protocol-handling) | 91 or later | Windows |  
+| [Window Controls Overlay for Installed Desktop Web Apps](#window-controls-overlay-for-installed-desktop-web-apps) | 91 or later | Windows |  
+| [Run on OS Login](#run-on-os-login) | 88 or later | All |  
+| [Shortcuts](#shortcuts) | 87 or later | All |  
+| [File Handling](#file-handling) | 83 or later | All Desktop|  
+
+## Features Coming Soon  
+
+This table includes features that are work in progress that may come out in a future release for testing.  
+
+| Feature | Platform |  
+|:--- |:--- |   
+| [URL Link Handling](#url-link-handling) | Windows |  
+| [Window Controls Overlay for Installed Desktop Web Apps](#window-controls-overlay-for-installed-desktop-web-apps) |  macOS, Linux |  
+| Improvements to App Management| All |  
+| Support for Desktop Share Target| Windows 10 |  
+
```

Comment:
Protocol handlers for macOS and Linux is coming soon as well.

Reply:
Thanks. Added an entry to the future table.